tests/test/test_main.py

```diff
@@ -0,0 +1,78 @@
+import sys
+import os
+sys.path.append('/Users/davidparker/Documents/Flywheel/SSE/MyWork/Gears/WSI/fwgear-wsi-to-dicom-converter')
```

Comment:
Maybe use relative path so that someone else could easily run this test without modifying the file

Reply:
actually very important, thanks